### DISCUSSION TITLE
How to move navigation order?

### DISCUSSION THREAD

**Discussion by tasanyphy01773** [alshedivat/al-folio#1196] [Q&A]
![image](https://user-images.githubusercontent.com/68806666/220067479-cc277988-332c-4297-a2bf-bb393a7ba7fe.png)
How to move the tab `Research` before `Blog`?

**Comment by riasat-sheikh** [comment]
Inside the  `_includes/header.html` file move the below mentioned section

```html
{% if site.blog_nav_title %}
<!-- Blog -->
<li class="nav-item {% if page.url contains 'blog' %}active{% endif %}">
  <a class="nav-link" href="{{ '/blog/' | relative_url }}">{{ site.blog_nav_title }}
    {%- if page.url contains 'blog' -%}
    <span class="sr-only">(current)</span>
    {%- endif -%}
  </a>
</li>
{%- endif %}
```

to the end of `<!-- Other pages -->` section i.e., after the `{% endfor -%}` and before `{%- if site.enable_darkmode %}`

**Comment by ctjustinehui** [comment]
You should also be able to change the navigation order in your .md file under nav_order. Give Blog a larger number than Research.
